components/sidebar/sidebar_category_list/index.ts

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {connect} from 'react-redux';
+import {bindActionCreators, Dispatch} from 'redux';
+
+import {getOrderedChannelIds} from 'mattermost-redux/selectors/entities/channels';
+import {getSidebarPreferences} from 'mattermost-redux/selectors/entities/preferences';
+import {GlobalState} from 'mattermost-redux/types/store';
+import {GenericAction} from 'mattermost-redux/types/actions';
+
+import SidebarCategoryList from './sidebar_category_list';
+
+// TODO: temp typing until we fix redux
+function mapStateToProps(state: GlobalState & {views: any}) {
+    // TODO: temp
+    const sidebarPrefs = getSidebarPreferences(state);
+    const lastUnreadChannel = state.views.channel.keepChannelIdAsUnread;
+    const orderedChannelIds = getOrderedChannelIds(
```

Comment:
Nothing to change right now, but we'll definitely be replacing this selector once I get to it. I'll aim to make its replacement return data of this same form, but that's something that might be changing.

Reply:
It can be a different form, this was just to get the sidebar working. we can coordinate on what's piped where once you're at that point.